libminifi/src/utils/LineByLineInputOutputStreamCallback.cpp

```diff
@@ -67,7 +67,7 @@ void LineByLineInputOutputStreamCallback::readLine() {
   if (end_of_line != input_.end()) { ++end_of_line; }
 
   current_line_ = next_line_;
-  next_line_ = utils::span_to<std::string>(gsl::make_span(&*current_pos_, &*end_of_line).as_span<char>());
+  next_line_ = utils::span_to<std::string>(utils::as_span<char>(std::span(&*current_pos_, &*end_of_line)));
```

Comment:
I think we could remove the `&*`s:
```suggestion
  next_line_ = utils::span_to<std::string>(utils::as_span<char>(std::span(current_pos_, end_of_line)));
```
as there is a `span` constructor which takes a pair of iterators

Reply:
I had to revert this as there doesn't seems to be an iterator overload constructor in the macos ci runner